docs/src/webview2.md

```diff
@@ -0,0 +1,128 @@
+---
+id: webview2
+title: "WebView2"
+---
+
+The following will explain how to use Playwright with [Microsoft Edge WebView2](https://docs.microsoft.com/en-us/microsoft-edge/webview2/). WebView2 is a WinForms control, which will use Microsoft Edge under the hood to render web content. It is a part of the Microsoft Edge browser and is available on Windows 10 and Windows 11. Playwright can be used to automate WebView2 applications and can be used to test web content in WebView2. For connecting to WebView2, Playwright uses [`method: BrowserType.connectOverCDP`] which connects to it via the Chrome DevTools Protocol (CDP).
+
+## Overview
+
+A WebView2 control can be instructed to listen on CDP by setting either the `WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS` environment variable with `--remote-debugging-port=9222` or call [EnsureCoreWebView2Async](https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.wpf.webview2.ensurecorewebview2async?view=webview2-dotnet-1.0.1343.22) with the `--remote-debugging-port=9222` argument. This will start the WebView2 process with the Chrome DevTools Protocol enabled.
+
+```txt
+await this.webView.EnsureCoreWebView2Async(await CoreWebView2Environment.CreateAsync(null, null, new CoreWebView2EnvironmentOptions()
+{
+  AdditionalBrowserArguments = "--remote-debugging-port=9222",
+})).ConfigureAwait(false);
+```
+
+Once your application with the WebView2 control is running, you can connect to it via Playwright:
+
+```js
```

Comment:
the language types are not shown in rendered version.

Reply:
What do you mean with that? Or just a note for future reviewers?